src/runtime/inspector_client.cpp

```diff
@@ -311,3 +330,384 @@ void TrInspectorClient::onReset()
   DEBUG(LOG_TAG_ERROR, "Request reset");
   end();
 }
+
+bool TrInspectorClient::tryUpgradeToWebSocket()
+{
+  // Check if this is a WebSocket upgrade request
+  bool hasUpgrade = false;
+  bool hasConnection = false;
+  bool hasWebSocketKey = false;
+  std::string webSocketKey;
```

Comment:
```suggestion
  string webSocketKey;
```
At the head of this file, `using namespace std` has been used.